atak-civ/plugin-examples/plugintemplate/app/src/main/java/com/atakmap/android/plugintemplate/PresetTimerListAdapter.java

```diff
@@ -135,6 +135,8 @@ public void onClick(View v) {
                 int pos = preset_timers.indexOf(currentTimer);
                 preset_timers.remove(currentTimer);
                 notifyItemRemoved(pos);
+                PluginTemplateDropDownReceiver.presets = preset_timers;
+                PluginTemplateDropDownReceiver.writePresetsToJSON(preset_timers);
```

Comment:
There is a TODO in line 134 that I had put for deleting timers. Now that deleting presets is done, can you delete the TODO?